### PR TITLE
Write doc for updating listener error

### PR DESCRIPTION
**Pull request by zhaoqin-github**

### CODE REVIEW

**Comment by zhaoqin-github** [comment]
@jputrino In this doc change the url link of OpentStack patch, https://review.openstack.org/#/c/518455/ , includes a pound sign. Unfortunately this link can not pass sphinx link check. I did not find a way to workaround this problem. Do you know how to fix it? Thanks in advance!

```
(line    6) broken    https://review.openstack.org/#/c/518455/ - Anchor '/c/518455/' not found
```

**Review by Niklaus-xie — COMMENTED** [review]
when a user attempts to update a listener with TERMINATED_HTTPS protocol  (or update a TERMINATED_HTTPS listener).  this seems better to me . but the original one is also ok to me .

**Comment by jputrino** [comment]
@zhaoqin-github You can add `linkcheck_ignore = True` to the conf.py file to resolve the issue with the link checker: http://www.sphinx-doc.org/en/master/usage/configuration.html#options-for-the-linkcheck-builder

File: docs/update-listener-error.rst [hunk 1]
```diff
@@ -0,0 +1,6 @@
+.. _update-listener-error:
+  
+Update the listener of a Terminated HTTPS Load Balancer
```
Comment (jputrino) [inline docs/update-listener-error.rst]: I don't think this requires its own document. It's fine to include this in the docs/README.

Agree with @Niklaus-xie on the suggested wording change.

**Review by jputrino — APPROVED** [review]

**Comment by zhaoqin-github** [comment]
@Niklaus-xie Can you please review the new doc? Thanks!

**Pull request merged**